e2e/vmss.go

```diff
@@ -269,6 +269,10 @@ func extractLogsFromVMWindows(ctx context.Context, s *Scenario) {
 	s.T.Logf("run command executed successfully: %v", runCommandResp)
 
 	s.T.Logf("uploaded logs to %s", blobUrl)
+	s.T.Logf("view storage account in Azure portal: https://portal.azure.com/?feature.customportal=false#@microsoft.onmicrosoft.com/resource/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Storage/storageAccounts/%s/overview",
```

Comment:
Oooh, if you add the cool ado stuff at the beginning of the line, it'll appear in the ADO warnings area - which means we won't have to hunt log files.

		fmt.Printf("##vso[task.logissue type=warning;]%s", message)